app/models/education_benefits_claim.rb

```diff
@@ -121,7 +121,27 @@ def form_matches_schema
     return unless form_is_string
     return unless FORM_TYPES.include?(form_type)
 
-    errors[:form].concat(JSON::Validator.fully_validate(FORM_SCHEMAS[form_type], parsed_form))
+    errors[:form].concat(JSON::Validator.fully_validate(get_form_schema, parsed_form))
+  end
+
+  def get_form_schema
+    schema = FORM_SCHEMAS[form_type]
+
+    if is_1995?
+      schema = schema.deep_dup
+      required = []
+
+      required <<
```

Comment:
i think @jbalboni said the frontend can't use oneOf because it doesn't work well with react-jsonschema-form